sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1044,6 +1044,49 @@ export class BlobClient extends StorageClient {
    * @param {BlobDownloadOptions} [options] Optional options to Blob Download operation.
    * @returns {Promise<BlobDownloadResponseModel>}
    * @memberof BlobClient
+   *
+   * @example
```

Comment:
removing this `@example` tag, right?

@ramya-rao-a

Reply:
Ramya and I resolved to leave the @example tag in for now, as the rest of the package does this as well.  We will figure out what to do with `@example` vs markdown-body examples after GA, but it requires some more discussion and a more comprehensive review of all of our packages.